### PR TITLE
added linebreaks after username

### PR DESCRIPTION
**Pull request by Glassinatorn**
Fixed a bug where usernames where on the same line while viewing permissions, without any seperation. 
They are now on seperate lines

### CODE REVIEW

File: js/projects/home.js [hunk 1]
```diff
@@ -20,10 +20,10 @@ let loadAllProjects = async () => {
             for (let n = 0; n < json[i].access.length; n++) {
                 if (json[i].access[n].permission == "w") {
```
Comment (SpaceLenore) [inline js/projects/home.js:21]: wtf???

**Review by SpaceLenore — CHANGES_REQUESTED** [review]
uh, the if and else statement does the same thing (as highlighted by my comment) either they should do different thing or we're doing a unnecessary comarison. @Glassinatorn @JohanDjarvKarltorp

File: js/projects/home.js (hunk 1 above)
Comment (Glassinatorn) [reply js/projects/home.js:21]: have added span tags as seen in commit 8949364

Comment (SpaceLenore) [reply js/projects/home.js:21]: I will fix some css for it then

**Review by SpaceLenore — APPROVED** [review]
looks good :artificial_satellite:

**Pull request merged**